### PR TITLE
Update cl_base.lua

### PR DESCRIPTION
**Pull request by Flowrider**
Fixed Error Message RegisterKeyMapping for RedM

RegisterKeyMapping Error TxAdmin

### CODE REVIEW

**Comment by tabarra** [comment]
This is a weird issue, never happened before.  
I think this native used to be a no-op in previous client versions.  
Do you know if this bug prevents the tx menu from working in game?

**Comment by Flowrider** [comment]
The menu works perfectly in Game. You only see the error when connecting, which triggers my Monk :D

Actually, it's only an optical bug, at least so far, but it was easy and quick to disable with my change and shouldn't cause any problems.

I don't know from which version this error occurs, I'm currently on Canary, but probably also in beta. I have not tested the release.

**Comment by outsider31000** [comment]
> This is a weird issue, never happened before. I think this native used to be a no-op in previous client versions. Do you know if this bug prevents the tx menu from working in game?

 I made a pr to fix this some weeks ago as well

**Comment by outsider31000** [comment]
#928  specific here with other stuff in the pr too

that native is not available in RedM hence why it errors.

File: resource/menu/client/cl_base.lua [hunk 1]
```diff
@@ -97,11 +97,13 @@ RegisterNetEvent('txcl:setAdmin', function(username, perms, rejectReason)
     debugPrint("^2[AUTH] logged in as '" .. username .. "' with perms: " .. json.encode(perms or "nil"))
     menuIsAccessible = true
     menuPermissions = perms
-    RegisterKeyMapping('txadmin', 'Menu: Open Main Page', 'keyboard', '')
-    RegisterKeyMapping('txAdmin:menu:openPlayersPage', 'Menu: Open Players page', 'KEYBOARD', '')
-    RegisterKeyMapping('txAdmin:menu:noClipToggle', 'Menu: Toggle NoClip', 'keyboard', '')
-    RegisterKeyMapping('txAdmin:menu:togglePlayerIDs', 'Menu: Toggle Player IDs', 'KEYBOARD', '')
-    RegisterKeyMapping('txAdmin:menu:tpToWaypoint', 'Menu: Teleport to Waypoint', 'KEYBOARD', '')
+    if not IS_REDM then
```
Comment (tabarra) [inline resource/menu/client/cl_base.lua]: Better to use `IS_FIVEM` instead.

Comment (Flowrider) [reply resource/menu/client/cl_base.lua]: Done and works.

**Comment by tabarra** [comment]
Just to add context, this actually breaks the menu a little bit.  
User report I got:
> when I give permissions to any one all admin have to relog in to have access to noclip/ tp. baiscily all the features except the TXAdmin tab

Since it fails on the key mapping command, it never reaches the setPermissions.
image

**Pull request merged**